src/components/spaces/helpers/CreateSpaceButton.tsx

```diff
@@ -3,22 +3,25 @@ import { ButtonProps } from 'antd/lib/button'
 import { ButtonLink, IconLink } from 'src/components/utils/CustomLinks'
 import { BareProps } from 'src/components/utils/types'
 
-const newSpacePath = '/spaces/new'
-const title = 'Create space'
+const getNewSpacePath = (asProfile?: boolean) =>
+  '/spaces/new' + (asProfile ? '?as-profile=true' : '')
+const getTitle = (asProfile?: boolean) => `Create ${asProfile ? 'profile' : 'space'}`
```

Comment:
You use the code `asProfile ? 'profile' : 'space'` in lot of places, so it is not good for long support, better if you create small function for it

Reply:
After I edit the url links, and refactor the one in edit space, now this type of code is only in two places. I think it's not necessary to create a wrapper for the wording in this case